.github/workflows/require-qc-label.yml

```diff
@@ -0,0 +1,39 @@
+name: QC Confirmation Required
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize, labeled, unlabeled]
+
+jobs:
+  enforce-qc-label:
+    name: QC Confirmation Check
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Check for 'qc-reviewed ✅' label
+        id: check_qc_label
+        continue-on-error: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ISSUE_URL: ${{ github.event.pull_request.issue_url }}
+        run: |
+          echo "🔍 Checking if 'qc-reviewed ✅' label is present..."
+
+          LABELS=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" "$ISSUE_URL/labels" | jq -r '.[].name')
+
+          echo "Labels found: $LABELS"
+          if echo "$LABELS" | grep -q "qc-reviewed ✅"; then
+            echo "✅ Label found. QC check confirmed."
+            exit 0
+          else
+            echo "⚠️ Label 'qc-reviewed ✅' is missing. Please add it after reviewing QC reports."
+            exit 1
+          fi
+
+      - name: Make a comment if "Continue on Error" check failed
+        uses: mainmatter/continue-on-error-comment@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          outcome: ${{ steps.check_qc_label.outcome }}
+          test-id: PR is missing the 'qc-reviewed ✅' label
```

Comment:
Hrm. This GitHub Actions workflow is cute, but it doesn't refresh itself when the label is added, so the "Check failed" comment never goes away.